CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.3.0 / 2016-06-07
+* [CHANGE] Push now rejects improper and reserved labels.
+* [CHANGE] Required labels flag removed.
+* [BUGFIX] Docker image actually works now.
+* [ENHANCEMENT] Converted to Promu build process.
```

Comment:
Could even be listed as change, as the structure of the release tarball changes.